pkg/private/tar/build_tar.py

```diff
@@ -87,7 +88,9 @@ def normalize_path(self, path: str) -> str:
     # We silently de-dup that. If people come up with a real use case for
     # the /a/b/a/b/rest... output we can start an issue and come up with a
     # solution at that time.
```

Comment:
Might want to update this comment since we now do allow it if the option is specified